src/main/resources/upgrade-files/post-upgrade/sql/UAF-4199Remove_Discontinued_Outbound_Messaging_MOD.sql

```diff
@@ -0,0 +1,20 @@
+--=======================================================================================================================================
```

Comment:
Last fall we had to add them to the kfsdbupgrade.properties file.  However for the past month or so the code has been changed, by Natalia I think, to scan the post upgrade folder and pickup what ever is there.  So no I don't believe we need to add it to .properties anymore. Thanks.